index.html

```diff
@@ -0,0 +1,311 @@
+            >
+          </div>
+        </div>
+        
+      </div>
+      <div class="about-me">
+        <section class="about-me-section">
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <h1 class="form-title">I'm always interested in hearing about new projects,<br>so if you'd like to chat please get in touch.</h1>
+    <footer>
+      <form class="form">
+        <label for="name">Name:</label><br>
+        <input id="name" type="text" name="name" maxlength="30"><br>
+        <label for="email">Email:</label><br>
+        <input id="email" type="email" name="email"><br>
+        <label for="message">Message:</label><br>
+        <input id="message" type="text" name="message"  maxlength="500"><br>
```

Comment:
- [ ] [OPTIONAL]  The design of your form inputs is well done! 👍  However they would look _even_ better if the [given](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1) design is followed.

- This is the design on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1):
  
![image](https://user-images.githubusercontent.com/57421823/147368587-1906c622-12d1-4612-872b-2da0cc7088e7.png)

- This is the form on your submission:
  
![image](https://user-images.githubusercontent.com/57421823/147368682-a5bf67e2-ebed-4e7f-9c8d-831cf9718a47.png)


I would really recommend you to optionally go ahead and implement it.  It would hugely improve the form's overall look. 💯